Doc/library/unittest.rst

```diff
@@ -46,6 +46,8 @@ test runner
    a textual interface, or return a special value to indicate the results of
    executing the tests.
 
+Output is in color by default and can be
```

Comment:
```suggestion
Output is colorized by default on Python 3.14 and newer. This can be
```

Reply:
Or perhaps?
```suggestion
.. versionadded:: 3.14
   Output is colorized by default and can be
   :ref:`controlled using environment variables <using-on-controlling-color>`.
```

Reply:
Nice, that LGTM!

Reply:
And if so, move it under the `versionchanged` at "Command-Line Interface"?

https://docs.python.org/3/library/unittest.html#command-line-interface

Reply:
That also makes sense, yes!